Code/Editor/IEditorImpl.cpp

```diff
@@ -238,7 +238,16 @@ void CEditorImpl::LoadPlugins()
     AZ::ComponentApplicationBus::BroadcastResult(exeFolder, &AZ::ComponentApplicationRequests::GetExecutableFolder);
 
     QDir testDir;
+#if defined(AZ_PLATFORM_MAC)
```

Comment:
This probably needs to go into stab branch too.

Reply:
yup I'm in the process of setting up a PR

Reply:
This should use grab the Binary Path from using the same logic as the [DynamicModuleHandle_UnixLike.cpp](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/Platform/Common/UnixLike/AzCore/Module/DynamicModuleHandle_UnixLike.cpp#L99-L101) which is used to locate runtime output folder when loading Gems.